endpoints.tf

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (C) 2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ */
+
+resource "aws_vpc_endpoint" "remote_metastores" {
```

Comment:
Why are we creating vpc endpoint for WaggleDance?

Reply:
how are we going to differentiate `remote_metastore` of different region and manage?
vpc_endpoint does not work cross-region as of now

Reply:
this is intended for services running in same region. 
terraform deployment will fail if you specify resources in another region.